doc/source/dev/contributor/quickstart_ubuntu.rst

```diff
@@ -2,9 +2,9 @@
 
 .. _quickstart-ubuntu:
 
-=================================================
-Development Environment Quickstart Guide (Ubuntu)
-=================================================
+=======================================================
+Development Environment Quickstart Guide (Ubuntu 16.04)
+=======================================================
```

Comment:
Might work with other versions, but this procedure has not failed for me in 16.04 on several machines (native linux, Amazon EC2 instance, within Docker container on Windows, WSL on Windows).